src/pages/experiments/[experimentId]/plots-and-tables/volcano/index.jsx

```diff
@@ -76,8 +75,8 @@ const VolcanoPlot = ({ experimentId }) => {
       controls: [{
         name: 'volcanoDimensions',
         props: {
-          xMax: Math.round(spec.xMax),
-          yMax: Math.round(spec.maxNegativeLogpValue) + 2,
+          xMax: 20,
```

Comment:
Why is this value now hardcoded at 20?

Reply:
I explained this in the commit description; basically, `xMax` is almost the same as what auto sets, which meant the slider offered basically no adjustment with it as a maximum value. Maybe it would be better to set it here to `xMax * 10`.